### PR TITLE
test: sequential tests for ember

### PR DESCRIPTION
**Pull request by brendaHuang-2023**
sequential tests for ember

### CODE REVIEW

**Comment by brendaHuang-2023** [comment]
This PR needs the code changes from other test PRs, mainly the added dependencies in test-support/fixtures/ember/addon-template project. Will merge this PR after the others have been merged.

**Comment by brendaHuang-2023** [comment]
THIS PR WILL BE THE LAST OF ALL TEST PRS TO MERGE. Before it's merged, I will need to retrigger the CI tests even though it may have passed earlier.

File: packages/cli/test/commands/migrate/sequential.ember-addon.test.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+const logger = createLogger({
+  transports: [new transports.Console({ format: format.cli() })],
+});
+
+const projects = {
+  emberAddon: getEmberAddonProject(),
+};
+
+describe('Task: sequential - ember addon', () => {
```
[the remark this answers is not in the document]
Comment (lynchbomb) [reply packages/cli/test/commands/migrate/sequential.ember-addon.test.ts:31]: ^^^ and the delta from this test vs the other migrate test eg. e2e config, e2e ...

File: packages/cli/test/commands/migrate/sequential.ember-app.test.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          analyzeTask(options),
+          sequentialTask(options, logger, previousRuns),
+        ];
+
+        await listrTaskRunner(tasks);
+        expect(output).toMatchSnapshot();
+
+        const appPath = resolve(project.baseDir, 'app');
+        const appFiles = readdirSync(appPath);
+        expect(appFiles).toContain('app.ts');
```
Comment (lynchbomb) [inline packages/cli/test/commands/migrate/sequential.ember-app.test.ts:99]: these are only checking that `git mv` worked, which isn't super helpful. this test should be focused on running the migration **sequentially**. there should be a delta between running the migration as a single pass (e2e tests) vs in sequence (this test) and then testing against that delta

**Review by lynchbomb — CHANGES_REQUESTED** [review]
lets be sure the tests around sequential migration are focused on just that. eg

- validate the project was partially migrated
- ... run sequential migration on remainder
- validate the entire project is now migrated

//////

- validate the project was partially migrated
- ... run sequential migration on 1 of N child packages
- validate this child package is migrated
- N of N child packages are now migrated

**Pull request closed without merging**